modules/external_petsc_solver/test/tests/kernels/simple_diffusion/simple_diffusion.i

```diff
@@ -0,0 +1,44 @@
+[Mesh]
```

Comment:
If you have a real test, feel free to delete this simple diffusion test. It's just nice to have a single placeholder.